lib/response.ex

```diff
@@ -0,0 +1,301 @@
+defmodule Solicit.Response do
```

Comment:
Thoughts on consolidating the package's public API (this looks like the majority of what consumers will be using) to the top-level `Solicit` module?

@mattludwigs posted this (warning: long) article on Elixir API design on Slack a couple months ago https://aaronrenner.io/2019/09/18/application-layering-a-pattern-for-extensible-elixir-application-design.html

Reply:
I think that could be a great API design. I think it may be worthwhile to get opinions on Monday as well.